tests/test_augmentation.py

```diff
@@ -49,3 +166,26 @@ def test_save_load(data_format, save_format):
         layer_class=ChannelSwap,
         training=None,
     )
+
+
+@pytest.mark.parametrize('data_format', ['default', 'channels_first', 'channels_last'])
+@pytest.mark.parametrize('save_format', ['h5'])
```

Comment:
I couldn't save the layer using 'tf' save_format. Still working into it.  The problem is that something happens when trying to save the model with this format, because when I was debugging, I found SpecAugment was receiving the `training`parameter to True, even if its default is None.